src/Geometry.cpp

```diff
@@ -12,16 +12,13 @@ namespace {
 
 Geometry::Geometry( const Geometry& g )
     : vertexBuffer( g.vertexBuffer )
-    , textureBuffer( g.textureBuffer )
     , colourBuffer( g.colourBuffer )
     , normalBuffer( g.normalBuffer )
     , faceBuffer( g.faceBuffer )
     , vertexArray( g.vertexArray )
-    , renderMode( g.renderMode )
-    , program( g.program )
-    , programOutline( g.programOutline )
     , vertices( g.vertices )
     , colours( g.colours )
+    , renderMode( g.renderMode )
```

Comment:
This will result in a compiler warning. I fixed this before, I think you merged wrong.